README.md

```diff
@@ -33,17 +33,19 @@ defmodule Resampling.Pipeline do
   @impl true
   def handle_init(_) do
     children = [
-      file_src: %File.Source{location: "/tmp/some_samples.raw"},
+      file_src: %File.Source{location: "/tmp/input.raw"},
       converter: %Converter{
         input_caps: %Raw{channels: 2, format: :s24le, sample_rate: 48_000},
         output_caps: %Raw{channels: 2, format: :f32le, sample_rate: 44_100}
       },
-      file_sink: %File.Sink{location: "/tmp/out.raw"},
+      file_sink: %File.Sink{location: "/tmp/output.raw"},
+    ]
+
+    links = [
+      link(:file_src) 
+      |> to(:converter),
+      |> to(:file_sink)
```

Comment:
Honestly I didn't test it, but probably I should?

Reply:
I would be great ;) you can check if there's already a similar pipeline in tests

Reply:
Syntax was almost correct but I am not sure what we expect as output. Checking resulting file with `ffprobe` gives me this
```
Input #0, amrnb, from 'beep-f32le-44kHz-stereo.raw':
  Duration: 00:27:01.98, bitrate: 11 kb/s
    Stream #0:0: Audio: amr_nb, 8000 Hz, mono, flt
```

I suspect it shouldn't be 8kHz and mono. It also isn't playable.

Reply:
A raw audio file contains just audio data without information about its shape, so FFmpeg tries to deduce it, but fails in this case. Try `ffplay -f f32le -ar 44100 -ac 2 output.raw`